components/ProfileMenu.vue

```diff
@@ -55,6 +56,14 @@ const saveAndCloseContentLanguageDialog = () => {
   )
     contentLanguageStore().contentLanguages = newLanguages;
 };
+
+const currentUser = useCurrentUser();
```

Comment:
Isn't it better to directly use the reactive ref you get from the `useCurrentUser` composable?
Since it is reactive, the UI will update once the data is fetched.

```ts
const { data: user } = useCurrentUser()
```

Reply:
that's a lot better. Thanks for pointing it out!